express-post-json/index.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const newArray = [];
+  for (const key in grades) {
+    newArray.push(grades[key]);
+  }
+  res.json(newArray);
+});
+
+app.use(express.json());
+
+app.post('/api/grades/', (req, res) => {
+  res.json(req.body);
+  grades[nextId] = req.body;
+  req.body.id = nextId;
+  nextId++;
+  res.status(201);
+  res.send(grades);
```

Comment:
The client should only receive the newly added grade in the response body for POST requests.